assets/czr/_dev/js/czr-control-dom_ready.js

```diff
@@ -119,6 +119,40 @@
                           return state.text;// optgroup different than google font
                         return '<span class="tc-select2-font">' + state.text + '</span>';
                     }
+
+                    /**
+                    * Dependency between the header layout and the menu position, when the menu style is Side Menu
```

Comment:
yeah I could (should).
But I wanted to fire this on ready too, and my latest tests reported that the section's awekening (as the dominus and servi are located in different sections) didn't work.
Maybe now is fixed, or I always did something wrong :D